src/utils/composeStores.js

```diff
@@ -1,11 +1,18 @@
-import mapValues from '../utils/mapValues';
-import pick from '../utils/pick';
+import mapValues from './mapValues';
+import pick from './pick';
+
+// UNDEF = undefined;
+const UNDEF = void 0;
```

Comment:
I think you can also invert the condition.
Instead of returning undefined in some cases, return *something* in other cases.

```js
function bad() {
  if (action.type === 'something') {
    return something;
  }
}
```